parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -1,3 +1,16 @@
 #!/bin/bash
 
-# Placeholder for future CIS-related shell utilities
\ No newline at end of file
+assignRootPW() {
+  grep '^root:[!*]:' /etc/shadow
+  if [ $? -eq '0' ] ; then
+    SALT=`openssl rand -base64 5`
+    SECRET=`openssl rand -base64 37`
+    CMD="import crypt, getpass, pwd; print crypt.crypt('$SECRET', '\$6\$$SALT\$')"
+    HASH=`python <<< "$CMD"`
```

Comment:
`python -c "$CMD"` is a more standard way to run a script in a string.

Reply:
This one we can do, proved that it's equivalent:

```
FrancisBookMS:aks-engine jackfrancis$ HASH=`python <<< "print('hello world')"`
FrancisBookMS:aks-engine jackfrancis$ echo $HASH
hello world
FrancisBookMS:aks-engine jackfrancis$ HASH=`python -c "print('hello world')"`
FrancisBookMS:aks-engine jackfrancis$ echo $HASH
hello world
```